release-notes/unreleased/LUN-442-back-button-in-word-list.yml

```
@@ -0,0 +1,6 @@
+issue_key: LUN-442
+show_in_stores: true
+platforms:
+  - android
+  - ios
+de: Button zum vorherigen Wort hinzugefügt
```

```suggestion
de: In der Wortliste kann jetzt zum vorherigen Wort navigiert werden
```
Is just a suggestion. It think the release notes should be understandable for the user what was improved. Maybe also simplify "navigiert" somehow

---

Great idea, thank you!